lib/membrane/rtp/outbound_tracking_serializer.ex

```diff
@@ -65,6 +72,8 @@ defmodule Membrane.RTP.OutboundTrackingSerializer do
 
   @impl true
   def handle_init(_ctx, options) do
+    Membrane.TelemetryMetrics.register(@frame_sent_telemetry_event, options.telemetry_label)
```

Comment:
You are ignoring RTCP packets sent by this element, a metric for that is missing